src/main/java/run/halo/injection/HtmlFooterProcessor.java

```diff
@@ -0,0 +1,41 @@
+package run.halo.injection;
+
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.thymeleaf.context.ITemplateContext;
+import org.thymeleaf.model.IModel;
+import org.thymeleaf.model.IProcessableElementTag;
+import org.thymeleaf.processor.element.IElementTagStructureHandler;
+import reactor.core.publisher.Mono;
+import run.halo.app.theme.dialect.TemplateFooterProcessor;
+
+
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class HtmlFooterProcessor extends AbstractHtmlProcessor  implements TemplateFooterProcessor {
```

Comment:
同时需要注意一下代码格式格式化问题，建议对这几个改动的类都执行一下代码格式化
```suggestion
public class HtmlFooterProcessor extends AbstractHtmlProcessor implements TemplateFooterProcessor {
```